examples/openai/env.example

```diff
@@ -0,0 +1,20 @@
+# Update this with your real OpenAI API key
+OPENAI_API_KEY=sk-YOUR_API_KEY
+
+# OTEL_EXPORTER_* variables are not required. If you would like to change your
+# OTLP endpoint to Elastic APM server using HTTP, uncomment the following:
+# OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:8200
+# OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+
+OTEL_SERVICE_NAME=openai-example
+
+# OpenAI instrumentation is experimental and disabled by default so enable it.
+OTEL_INSTRUMENTATION_OPENAI_CLIENT_ENABLED=true
+
+# Change to 'false' to hide prompt and completion content
+OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT=true
+# Change to affect behavior of which resources are detected. Note: these
+# choices are specific to the language, in this case Python.
+OTEL_EXPERIMENTAL_RESOURCE_DETECTORS=process_runtime,os,otel,telemetry_distro
```

Comment:
This isn't needed for Java, right? Therefore we should remove it

Reply:
good idea